v5/v5_stable.ino

```diff
@@ -1731,3 +1689,125 @@ String get_HTTP_string(String message) {
   }
   return tempStr;
 }
+
+bool newPackageSubscribed(String str) {
```

Comment:
- You have **not defined** these functions before void setup() _where other functions are defined_
- This function should also update a variable in SPIFFS

Reply:
- This function should also send message to user that his package is also understanded by module.